__tests__/app.test.js

```diff
@@ -44,3 +44,41 @@ describe("/api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("responds with status 200 and the requested article", () => {
+    return request(app)
+      .get("/api/articles/1")
```

Comment:
As this is a test for an actual individual article, then this test should test for the specifics, i.e. who is the author, what is the exact title, etc?

Because you're testing for a specific resource, you should test for the exact property values, rather than the 'shape' of the data.

Reply:
Great work! 

For future development, maybe have a look at [.toMatchObject( )](https://jestjs.io/docs/expect#tomatchobjectobject) and [expect.objectContaining( )](https://jestjs.io/docs/expect#expectobjectcontainingobject)